test/keboola/snapshots/serializelists/out/tables/adsets/1638290127158

```diff
@@ -42,7 +42,7 @@
 "23845136125380682","act_522606278080331","page_adsets","act_522606278080331",""
 "23845201552990682","act_522606278080331","page_adsets","act_522606278080331",""
 "23845270930970682","act_522606278080331","page_adsets","act_522606278080331",""
-"23845271377410682","act_522606278080331","page_adsets","act_522606278080331","{""event"":""IMPRESSIONS"",""interval_days"":30,""max_frequency"":6}"
+"23845271377410682","act_522606278080331","page_adsets","act_522606278080331","[{""event"":""IMPRESSIONS"",""interval_days"":30,""max_frequency"":6}]"
```

Comment:
predtym sa pole serializovalo po objektoch na kazdy riadok tabulky, zrovna v tych prikladoch je len jeden objekt v poli tak to neni uplne zrejme. Po novom sa serializuje cele pole tj nezacina s `{` ale `[{`